pkg/cli/admin/release/extract_tools.go

```diff
@@ -457,7 +457,10 @@ func (o *ExtractOptions) extractCommand(command string) error {
 		if len(missing) == 1 {
 			return fmt.Errorf("the image %q containing the desired command is not available", missing.List()[0])
 		}
-		return fmt.Errorf("some required images are missing: %s", strings.Join(missing.List(), ", "))
+		if len(missing) > 1 {
+			return fmt.Errorf("some required images are missing: %s", strings.Join(missing.List(), ", "))
+		}
+		return fmt.Errorf("the command '%s %s' is not available for architecture %s", command, o.From, runtime.GOARCH)
```

Comment:
when I build the oc with this pr , the output is :
./oc adm release extract -a  config --command=oc  quay.io/openshift-release-dev/ocp-release:4.13.0-ec.0-x86_64  --to=/root/1122/
error: the command 'oc quay.io/openshift-release-dev/ocp-release:4.13.0-ec.0-x86_64' is not available for architecture arm64

I think we want to print like : the command 'oc' is not available for architecture arm64 on [quay.io/openshift-release-dev/ocp-release:4.13.0-ec.0-x86_64](http://quay.io/openshift-release-dev/ocp-release:4.13.0-ec.0-x86_64)

Reply:
Yeah that sounds better. It's been updated as such.
```
[bcox@dev-002 oc]$ ./oc adm release extract -a ~/all-the-pull-secrets.json --command=oc quay.io/openshift-release-dev/ocp-release:4.13.0-ec.0-x86_64 --to=/tmp
error: the command 'oc' is not available for linux/arm64 on quay.io/openshift-release-dev/ocp-release:4.13.0-ec.0-x86_64
```